### ISSUE TITLE
Home Page Authentication App Auth Timeout

### ISSUE THREAD

**Issue by anywhichway** [feathersjs/feathers#584]
Follow the instructions on https://feathersjs.com/ to create first real-time app.

Application starts up fine and loads page. Opening the developer console as instructed shows that the auth attempt is timing out. Tried using Chrome and Firefox. Hard to follow the transpiled code in the browser debugger, so not sure what to do next.

Do you have a GitHub repo I could pull the example from to verify?

### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))

- [ ] Tell us what broke. The more detailed the better.
- [ ] If you can, please create a simple example that reproduces the issue and link to a gist, jsbin, repo, etc.

### Expected behavior
Authentication should be successful

### Actual behavior
Timeout by auth promise

Server Trace:
 engine setting new request for existing client +1ms
  engine:polling received "90:420["authenticate",{"strategy":"local","email":"admin@feathersjs.com","password":"admin"}]" +2ms
  engine:socket packet +2ms
  socket.io-parser decoded 20["authenticate",{"strategy":"local","email":"admin@feathersjs.com","password":"admin"}] as {"type":2,"nsp":"/","id":0,"data":["authenticate",{"strategy":"local","email":"admin@feathersjs.com","password":"admin"}]} +1ms
  socket.io:socket got packet {"type":2,"nsp":"/","id":0,"data":["authenticate",{"strategy":"local","email":"admin@feathersjs.com","password":"admin"}]} +2ms
  socket.io:socket emitting event ["authenticate",{"strategy":"local","email":"admin@feathersjs.com","password":"admin"}] +0ms
  socket.io:socket attaching ack callback to event +0ms
  socket.io:socket dispatching an event ["authenticate",{"strategy":"local","email":"admin@feathersjs.com","password":"admin"},null] +1ms
  engine:ws received "2probe" +417ms
  engine:ws writing "3probe" +0ms
  engine:socket writing a noop packet to polling for fast upgrade +102ms
  engine:polling writing "1:6" +3ms
  engine:ws received "5" +9ms
  engine:socket got upgrade packet - upgrading +2ms
  engine:polling closing +1ms
  engine:polling transport discarded - closing right away +0ms
  engine:ws received "2" +2s
  engine:socket packet +1ms
  engine:socket got ping +0ms
  engine:socket sending packet "pong" (undefined) +1ms
  engine:socket flushing buffer to transport +0ms
  engine:ws writing "3" +0ms

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**NodeJS version**:  v7.3.0

**Operating System**: Windows 64

**Browser Version**: Chrome 59 and Firefox 53

**React Native Version**: N/A

**Module Loader**: npm

**Comment by marshallswain** [comment]
You can check out the feathers-chat repo.  The public folder has example apps in a few different frameworks.
